### PR TITLE
Landing css cleanup

### PR DESCRIPTION
**Pull request by eshathakur**
for issue #121 , cleaned landing.css did minor changes , it was already ordered by specificity I just added and took out some comments

### CODE REVIEW

File: src/password/landing.css [hunk 1]
```diff
@@ -18,28 +21,24 @@ body {
     margin: 0;
 }
 
-/* Styling for the form */
 form {
     border: 2px solid var(--form-border-color);
```
Comment (shambhavi-20) [inline src/password/landing.css:25]: Makes these into variables

**Review by shambhavi-20 — CHANGES_REQUESTED** [review]
Could you make all the numbers and colors into variables? Also, have a comment before each tag explaining what it does like write what ".error" represents. Also, try to explain why a flex box is used. It is fine if that seems trivial.
If there is an h1 tag on the page, add:
```
h1 {
    font-weight: normal;
}

**Comment by eshathakur** [comment]
there was no h1 tag, but let me know if I should take out the comments that explain some of the selectors I thought weren't obvious

**Comment by shambhavi-20** [comment]
I dont know what happened but you a pushing a bunch of new files which have the same code as all the other files in the repo but just a "2" attached to their name. Can you fix this or mayne make a new pull request where you just push landing.css?

**Pull request closed without merging**